src/test/util.h

```diff
@@ -39,11 +41,34 @@ namespace openmldb {
 namespace test {
 
 using ::openmldb::codec::SchemaCodec;
-
 inline std::string GenRand() {
     return std::to_string(rand() % 10000000 + 1);  // NOLINT
 }
 
+class TempPath {
+ public:
+    TempPath() {
+        std::filesystem::path tmp_path = std::filesystem::temp_directory_path();
+        base_path_ = absl::StrCat(tmp_path.string(), "/openmldb_test", ::openmldb::test::GenRand());
+    }
+    ~TempPath() {
+        std::filesystem::remove_all(base_path_);
+    }
+    std::string GetTempPath() {
+        return absl::StrCat(base_path_, "/", ::openmldb::test::GenRand());
+    }
+    std::string GetTempPath(const std::string& prefix) {
+        return absl::StrCat(base_path_, "/", prefix, ::openmldb::test::GenRand());
+    }
+    std::string CreateTempPath(const std::string& prefix) {
+        std::string path = GetTempPath(prefix);
+        std::filesystem::create_directory(path);
```

Comment:
check ret value

Reply:
no need to check, program will core dump if create failed(not catch exception).